index.html

```diff
@@ -21,10 +21,18 @@ <h1 class="h-55">
       </h1>
       <nav class="navigation">
         <img
-          class="align-center"
+          class="align-center hide-desktop"
```

Comment:
is it BEM? 
Ich kann gerade nicht sehen was in der classe definiert ist, aber es klingt nicht nach utility class.

Reply:
It is a utility class to hide this element on desktop